utils/pdml_parser.py

```diff
@@ -67,7 +67,7 @@ def get_string(self, parsed):
         parsed = parsed.replace("0x", "")
         parsed = parsed.replace("\\x", "")
 
-        value = ""
+        value = six.binary_type(b"")
```

Comment:
The following `value.replace()` statements should be updated correspondingly. Otherwise, `TypeError` will occur in py3 env.

Reply:
Good call I'll update that